controllers/ebpf/agent_controller.go

```diff
@@ -355,6 +376,32 @@ func (c *AgentController) desired(ctx context.Context, coll *flowslatest.FlowCol
 
 	}
 
+	if helper.IsAgentFeatureEnabled(&coll.Spec.Agent.EBPF, flowslatest.EbpfManager) {
+		if !coll.Spec.Agent.EBPF.Privileged {
+			rlog.Error(fmt.Errorf("invalid configuration"), "To enable BPF Manager feature privileged mode needs to be enabled")
```

Comment:
this error/warning should also be added to the webhook: same as here https://github.com/netobserv/network-observability-operator/blob/main/apis/flowcollector/v1beta2/flowcollector_validation_webhook.go#L101-L103